### PR TITLE
rabbitMQ - wait for container to come up for 60 seconds

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/Shippable/admiral/issues/1120

tested by clean installing admiral and initializing and re- initializing the services
.

### CODE REVIEW

File: common/scripts/Ubuntu_14.04/installMsg.sh [hunk 1]
```diff
@@ -18,40 +18,45 @@ start_rabbitmq() {
...
   echo "Enabling rabbitmq management plugin"
   /usr/sbin/rabbitmq-plugins enable rabbitmq_management
 }
 
 check_rabbitmq_management_plugin() {
-  if ! nc -vz $MSG_HOST $ADMIN_PORT &>/dev/null; then
-    echo "Port $ADMIN_PORT not available for msg"
-    exit 1
-  fi
+  __check_service_connection "$MSG_HOST" "$AMQP_PORT" "MSG_ADMIN" "$TIMEOUT"
```
Comment (ric03uec) [inline common/scripts/Ubuntu_14.04/installMsg.sh]: - wrong port.
- add echo to say whats going on

File: common/scripts/Ubuntu_14.04/installMsg.sh [hunk 2]
```diff
@@ -18,40 +18,46 @@ start_rabbitmq() {
...
   echo "Enabling rabbitmq management plugin"
   /usr/sbin/rabbitmq-plugins enable rabbitmq_management
 }
 
 check_rabbitmq_management_plugin() {
-  if ! nc -vz $MSG_HOST $ADMIN_PORT &>/dev/null; then
-    echo "Port $ADMIN_PORT not available for msg"
-    exit 1
-  fi
+  echo "Checking rabbitmq admin on $MSG_HOST:$AMQP_PORT"
```
Comment (ric03uec) [inline common/scripts/Ubuntu_14.04/installMsg.sh]: wrong port... again. dude, check this stuff before sending prs

Comment (niranjan92) [reply common/scripts/Ubuntu_14.04/installMsg.sh]: @ric03uec sorry, fixed the port in logs

**Comment by ric03uec** [comment]
was rabbitmq initialized and reinitialized on a remote host to test this?

**Comment by niranjan92** [comment]
@ric03uec  yes

**Pull request merged**